namespace TestTryCatchTwoExceptions is
    use Std = IO.Std;

    class Main is
        entry() static is
            @IL.output("il.out")
            try
            // FIXME: scope of ex1 and ex2 should be within their
            // own catch clause only
            catch ex1: System.Exception
            catch ex2: MyException
            yrt
        si
    si

    class MyException: System.Exception is  
    si  
si